application/templates/application/application-form.html

```diff
@@ -6,11 +6,14 @@
 {{ form.media.css }}
 
 <div style="margin-left:20%; height: 90%; margin-bottom: 5%; width:50%; margin-top: 2%;">
+    {% if error_count_app %}
```

Comment:
Maybe we can have the same design as the "empty-list (broken)" for high school and admin staff here as well, handling this message and providing a bit more information to the user